lib/msal-browser/src/event/EventMessage.ts

```diff
@@ -3,7 +3,7 @@
  * Licensed under the MIT License.
  */
 
-import { AuthenticationResult, AuthError, EndSessionRequest } from "@azure/msal-common";
+import { AuthenticationResult, AuthError, CommonEndSessionRequest } from "@azure/msal-common";
```

Comment:
Q: Why is the `EventPayload` based on `msal-common` `EndSessionRequest` instead of `msal-browser`?

Reply:
Not sure, looks like it should be able to use the one from browser. @jo-arroyo can confirm.

Reply:
Hmm... must have been overlooked. Should be `EndSessionRequest` from `msal-browser` instead.

Reply:
Great! Can change this and `angular` dependency can be thrown out. Thanks.